src/js/media-store/util.ts

```diff
@@ -2,6 +2,10 @@ import { TextTrackKinds, TextTrackModes } from '../constants.js';
 import { getTextTracksList, updateTracksModeTo } from '../utils/captions.js';
 import { TextTrackLike } from '../utils/TextTrackLike.js';
 
+export const isMediaPlaying = (media) => {
+  return !media.paused && !media.ended && media.readyState > 2;
```

Comment:
Do we need this? None of these should ever be false when the `playing` event is fired.

Reply:
true, good call. I later added the `playing` event.type check and didn't realize